MSAL/src/MSALPublicClientApplication.m

```diff
@@ -97,6 +97,7 @@ @interface MSALPublicClientApplication()
 
 @property (nonatomic) MSIDDefaultTokenCacheAccessor *tokenCache;
 @property (nonatomic) MSALPublicClientApplicationConfig *internalConfig;
+@property (nonatomic) MSIDAccountMetadataCacheAccessor *accountMetadataCache;
```

Comment:
why do we duplicate it here?

Reply:
let's remove (internal) from the +internal header, thus removing duplicate